apps/src/storage/dataBrowser/DataLibraryPane.jsx

```diff
@@ -0,0 +1,24 @@
+import Radium from 'radium';
+import React from 'react';
+
+const styles = {
+  container: {
+    display: 'block',
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    width: 270,
+    boxSizing: 'border-box',
+    borderRight: '1px solid gray',
+    overflowY: 'auto',
+    padding: 10,
+    paddingRight: 0 // setting this to 0 allows 2 columns with the potential scrollbar on Windows
```

Comment:
In which cases is this needed? I pulled the branch really quick to test it out, and disabling this didn't cause a difference on IE or Chrome. Does this have a bigger effect when there is scrollable data?

Reply:
I think you might also be able to go without `display: 'block'`

Reply:
Would it make more sense to make the width 250 and go without the border-box? Not sure which is clearer, but it's an option.